.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           name: Install Python dependencies
           command: |
             pip --version
-            pip install --progress-bar off -r requirements/doc-requirements.txt pytest pytest-cov plotly .[gateway]
+            pip install --progress-bar off -r requirements/doc-requirements.txt pytest pytest-cov plotly .[gateway] ipywidgets
```

Comment:
Do we need `ipywidgets` when building docs?

Reply:
Yes, otherwise it fails

Reply:
Can you share a failed build URL? I want to see why it failed.

Reply:
https://app.circleci.com/pipelines/github/mlflow/mlflow/24838/workflows/9265b87f-2036-41fb-88c7-d97cce48a2fb/jobs/58568

Reply:
Can `tqdm.auto` solve this issue?

Reply:
Yes! Updated!